AEGIS_25.2_training_sample_readme.txt

```diff
@@ -43,101 +44,57 @@ NOISE_MEAN: mean value of the pixels in the image that do not contain the galaxy
 NOISE_VARIANCE: variance of the pixel values for pixels in the image that do not contain the galaxy.
 
 
-(2) The files containing galaxy and PSF images are AEGIS_galaxy_images_I(V)_25.2_n?.fits and
-AEGIS_galaxy_PSF_images_I(V)_25.2_n?.fits. The number that goes in place of
-the "?" ranges from 0 to 29.
+(2) The files containing galaxy and PSF images are
+AEGIS_galaxy_images_[VI]_25.2_n?.fits. The number that goes in place of the "?"
```

Comment:
Ahh right.  I over-edited.  I meant to convert the I(V) format to [IV] format.  BTW - This is because [IV] is a regular expression, just like the '?' or a '*' wildcard.